src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -86,6 +86,15 @@ class CoreWorkerDirectTaskSubmitter {
   Status CancelRemoteTask(const ObjectID &object_id, const rpc::Address &worker_addr,
                           bool force_kill);
 
+  /// Check that the scheduling_key_entries_ hashmap is empty. Can be used at the end of
+  /// a unit test (or normal program) to check that we are not leaking memory
+  bool CheckNoSchedulingKeyEntries() const {
+    if (scheduling_key_entries_.size() != 0) {
+      RAY_LOG(INFO) << "size: " << scheduling_key_entries_.size();
+    }
+    return scheduling_key_entries_.size() == 0;
```

Comment:
You can also use `scheduling_key_entries_.empty()` here!

Reply:
Good call! Just changed this. I also removed the RAY_LOG(INFO) line